apps/partner2/stylesheets/overview.styl

```diff
@@ -28,7 +28,7 @@
       min-height 100px
       .partner-overview-section-title
         avant-garde-size('m-headline')
-    &[data-module=about] + [data-module=news]
+    &[data-module=news]
```

Comment:
We're currently fetching a max of three. https://github.com/artsy/force/blob/master/apps/partner2/components/news/view.coffee#L19 But there will definitely be cases when there are only one or two.